libr/debug/p/native/linux/linux_debug.c

```diff
@@ -611,6 +615,46 @@ RDebugPid *fill_pid_info(const char *info, const char *path, int tid) {
 	return pid_info;
 }
 
+RList *linux_pid_list(int pid, RList *list) {
+	list->free = (RListFree)&r_debug_pid_free;
+	DIR *dh;
+	struct dirent *de;
+	char *ptr, st, buf[1024], info[1024];
```

Comment:
isnt this 1024 handled by a constant somewhere? like PATHMAX or so?

Reply:
PATH_MAX is 4096 usually but it's probably wiser to switch to it since cmdline could get to that length.

Reply:
use the heap then, allocate once before the loop. huge heap sizes use to be a source for problems in random operating systems